app/org/sagebionetworks/bridge/services/ParticipantService.java

```diff
@@ -398,7 +398,7 @@ public void resendVerification(Study study, ChannelType type, String userId) {
             AccountId accountId = AccountId.forPhone(study.getIdentifier(), participant.getPhone());
             accountWorkflowService.resendVerificationToken(type, accountId);
         } else {
-            throw new UnsupportedOperationException("Channel type not implemented");
+            throw new BadRequestException("Invalid request to resend verification");
```

Comment:
Note: This will fail an integration test as before, this would silently "succeed" while doing nothing.